lib/versioning/index.ts

```diff
@@ -21,10 +21,29 @@ function get(versionScheme: string) {
     logger.debug('Missing versionScheme');
     return schemes.semver;
   }
-  const scheme = schemes[versionScheme];
+  let schemeName;
+  let schemeConfig;
+  if (versionScheme.includes(':')) {
+    const versionSplit = versionScheme.split(':');
+    schemeName = versionSplit.shift();
+    schemeConfig = versionSplit.join(':');
+  } else {
+    schemeName = versionScheme;
+  }
+  const scheme = schemes[schemeName];
   if (!scheme) {
     logger.warn({ versionScheme }, 'Unknown version scheme');
     return schemes.semver;
   }
+  if (schemeConfig) {
```

Comment:
New Idea: `configure` returns an `VersioningApi` object and we use that return value if `configure` exists.

So the Scheme can decide what to do with the config and how the object is constructed.